sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/encoders/EncoderResolutionSuite.scala

```diff
@@ -94,8 +94,8 @@ class EncoderResolutionSuite extends PlanTest {
         "details" -> (
           s"""
           |The type path of the target object is:
-          |- array element class: "scala.Long"
-          |- field (class: "scala.Array", name: "arr")
+          |- array element class: "long"
```

Comment:
This is a side effect of not having the same type information when creating the deserializer.